onnxruntime/core/providers/dml/OperatorAuthorHelper/OperatorHelper.cpp

```diff
@@ -931,8 +937,8 @@ namespace OperatorHelper
             {
                 ML_CHECK_VALID_ARGUMENT(m_split.size() == 0);
                 auto splitDimSize = inputDimensions.at(m_axis);
-                auto evenParts = static_cast<int>(ceil((double)splitDimSize / numOutputs));
-                m_split = std::vector<int>(numOutputs, evenParts);
+                auto evenParts = DivideRoundUp(splitDimSize, numOutputs);
+                m_split.resize(numOutputs, evenParts);
                 m_split.back() = static_cast<int>(splitDimSize - (numOutputs-1) * evenParts);
```

Comment:
```suggestion
                auto inputSizeAlongAxis = inputDimensions.at(m_axis);
                auto outputSizeAlongAxis = DivideRoundUp(inputSizeAlongAxis, numOutputs);
                m_split.resize(numOutputs, outputSizeAlongAxis);
                // Every output has the same size except potentially the last one, which may be smaller.
                m_split.back() = static_cast<int>(inputSizeAlongAxis - (numOutputs - 1) * outputSizeAlongAxis);
```
Number spacing.
Renamed input/output sizes for consistency with some other parts.